package.json

```diff
@@ -46,7 +46,7 @@
     "react-mounter": "^1.2.0",
     "react-select": "^1.0.0-rc.3",
     "react-tools": "^0.13.3",
-    "react-tooltip": "^3.3.0",
+    "react-tooltip": "^3.5.1",
```

Comment:
updating the package, or removing it?

Reply:
This is just a temporary update until it is removed in the tooltips branch